### ISSUE TITLE
NMMNHS Paleo - Media

### ISSUE THREAD

**Issue by Nicole-Ridgwell-NMMNHS** [ArctosDB/data-migration#226]
My recommendations for media formatting and naming

**Image Format**

1. All raw images, pre-stacking, pre-color calibration be archived in .dng format.
2. High resolution, edited images be archived in .tiff format.
3. Lower resolution images for upload to Arctos be converted to .jpg format.

**File Naming Format**

P-#####_FL_YYYY-MM-DD_#
FL is first and last initial

**Arctos Image Resolution**

150 dpi
I prefer not to go higher than that, just so if someone is using the images for something other than a quick reference they will be inclined to actually contact us for the higher resolution and we can track that usage.

Thoughts? @Jegelewicz @hannahcantrell 

@PaulaBarteau @lin-fred are you doing something similar?

**Comment by lin-fred** [comment]
@PaulaBarteau Let's discuss this in our meeting on Monday

**Comment by Jegelewicz** [comment]
@Nicole-Ridgwell-NMMNHS I believe you are the most experienced with imaging. I think it would be really helpful if you could help Paula and Hannah. I know they have access to manuals and books and can Google lots of stuff, but it seems like you could offer them first-hand knowledge and some first-person training and mentoring. I realize this will be a bit more difficult due to social distancing requirements, but it isn't impossible. When Paula or Hannah are in the museum with the equipment, perhaps you can schedule to communicate via Webex and walk them through a few specimens? It would probably be helpful to have Hannah in on the virtual meeting when Paula is in the collection and vice versa. In the long run, I think this will save time and reduce duplication of effort. Plus the three of you could write documentation together either using GitHub or GoogleDocs, whichever is easier to use for you all. We need to make sure we aren't doing this in silos!

**Comment by Jegelewicz** [comment]
One other thing I realized - all of the imaging documentation that I got from Yale and elsewhere is on the N drive and I did not copy it to my laptop. Did anyone else do this? If not, we should move those files to the GoogleDrive so that everyone can see them. Whoever goes in to the museum next, please do this for us!

**Comment by hannahcantrell** [comment]
I think that's a great idea Teresa. I know that I have been frazzled about trying to learn these processes in a timely manner so I can make sure to get protocols written, and also as much images taken as possible. It's been a slow moving project. I learn best and easiest in hands-on situations. I know right now it's hard to coordinate and also get in to the museum to work on this, but I think we need to be making it our highest priority right now so that Paula and I both can succeed.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
> I think it would be really helpful if you could help Paula and Hannah.

I agree. It will also probably be easier for me to pull this information together now that I have fewer distractions.

> also as much images taken as possible

The important thing right now is to focus on our protocols and how we can do things to take the best quality images we can. So you don't need to worry so much about # of specimens imaged.

I can write up a tip sheet, but I agree it might be best to do a video conference while you all are actually working on the photos. Hannah, go ahead and plan on coming in to take photos next Friday, and we'll do a video call. I will hopefully be able to come in on Wednesday or Thursday to make sure everything is accessible for you.

**Comment by PaulaBarteau** [comment]
Hi all,
Lindsey and I are drafting our imaging protocols in g-docs, mostly we are
working on different kinds of specimen handling and general photography
instructions. I'll send Hannah a copy, they're a little bit rough right now
and we need to work some things out but I intend to ask for feedback from
everyone next week after editing them a bit more. It would be very helpful
to get some definitive in-put on where/how we want to store copies for
Arctos and copies for Archiving. I've been saving stacks and stacked images
as Tiffs, and then saving the stacked images again as Jpegs. So far I
haven't paid much attention to dpi. I can start saving the raw stacks as
DNGs now that we have Light Room and figure out how to set a restraint on
the dpi.
Nicole, do you have a preference for Wednesday or Thursday to go into the
collection?

Thanks everybody!
Paula

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I expect dpi will be best set in photoshop. Once you save the edited high-resolution file in tif, you can save it a second time as jpg and it should give you compression options where you can set the dpi.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Just leaving this here for when we get photography back up and running. We can use this software to speed up photography of multipart specimens:
https://naturalhistorymuseum.github.io/inselect/
